src/utils/octet_string.hpp

```diff
@@ -25,6 +25,11 @@ class OctetString
     {
     }
 
+    const std::vector<uint8_t>& getData() const
```

Comment:
**data** is already present. Please remove this method.

Reply:
I believe I used this getter() as m_data was a private member of the Octetstring class and for extractSliceInfoAndModifyPdu(), I needed the data from NAS PDU. Please let me know, if there is a more suitable alternative.

Reply:
Yes `m_data `is private, however we have public `data()` function. Please use data() function instead.